test/suites/network/multi-nic.robot

```diff
@@ -0,0 +1,186 @@
+
+    # Rebooting MicroShift host restores the network configuration
+    [Teardown]    Run Keywords
+    ...    Reboot MicroShift Host
+    ...    Login Switch To IP1
+    ...    Wait For Healthy System
+
+
+*** Keywords ***
+Setup
+    [Documentation]    Test suite setup
+    Login MicroShift Host
+    Initialize Global Variables
+    Setup Suite With Namespace
+    Verify Multiple NICs
+    Wait For Healthy System
+
+Teardown
+    [Documentation]    Test suite teardown
+    Logout MicroShift Host
+
+Verify Multiple NICs
+    [Documentation]    Verifies that the host has two Ethernet network interfaces
+
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    ${NMCLI_CMD} | wc -l
+    ...    return_stdout=True    return_stderr=True    return_rc=True
+    Log    ${stderr}
+    Should Be Equal As Integers    ${rc}    0
+    Should Be Equal As Strings    ${stdout}    ${NICS_COUNT}
+
+Initialize Global Variables
+    [Documentation]    Initializes global variables.
+
+    Log    Host: ${USHIFT_HOST_IP1} ${USHIFT_HOST_IP2}
+    Should Not Be Empty    ${USHIFT_HOST_IP1}    USHIFT_HOST_IP1 variable is required
+    Should Not Be Empty    ${USHIFT_HOST_IP2}    USHIFT_HOST_IP2 variable is required
+    Initialize Nmcli Variables
+
+Initialize Nmcli Variables
+    [Documentation]    Initialize the variables on the host
+
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    ${NMCLI_CMD} | head -1
+    ...    return_stdout=True    return_stderr=True    return_rc=True
+    Log    ${stderr}
+    Should Be Equal As Integers    ${rc}    0
+    Set Suite Variable    \${NIC1_NAME}    ${stdout}
+
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    ${NMCLI_CMD} | tail -1
+    ...    return_stdout=True    return_stderr=True    return_rc=True
+    Log    ${stderr}
+    Should Be Equal As Integers    ${rc}    0
+    Set Suite Variable    \${NIC2_NAME}    ${stdout}
+
+Nmcli Connection Control
+    [Documentation]    Run nmcli connection command with arguments
+    [Arguments]    ${command}    ${conn_name}
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    nmcli connection ${command} ${conn_name}
+    ...    sudo=True return_stdout=False    return_stderr=True    return_rc=True
+    Log    ${stderr}
+    Should Be Equal As Integers    ${rc}    0
+
+Login Switch To IP
+    [Documentation]    Switch to using the specified IP for SSH connections
+    [Arguments]    ${new_ip}
+
+    IF    '${USHIFT_HOST}'!='${new_ip}'
+        Logout MicroShift Host
+        Set Global Variable    \${USHIFT_HOST}    ${new_ip}
+        Login MicroShift Host
+    END
+
+Login Switch To IP1
+    [Documentation]    Switch to using the first IP for SSH connections
+    Login Switch To IP    ${USHIFT_HOST_IP1}
+
+Login Switch To IP2
+    [Documentation]    Switch to using the second IP for SSH connections
+    Login Switch To IP    ${USHIFT_HOST_IP2}
+
+Verify Hello MicroShift LB
```

Comment:
This is basically `Load Balancer Smoke Test` from networking-smoke.robot.
Should we move `Verify Hello MicroShift LB` to `microshift-network.resource` and reuse in both places?